data/templates/https/nginx.default.j2

```diff
@@ -18,6 +18,8 @@ server {
     listen [::]:{{ port }} ssl;
 {% endif %}
 
+    server_tokens {{ 'on' if server_version_id is vyos_defined and server_version_id == 'enable' else 'off' }};
```

Comment:
With default disabled I actually ment this ;)
```suggestion
    server_tokens 'off';
```

Reply:
We do not use bool `yes|no, disable|enable, true|false`, and so on
+ I also prefer to disable the version as in the suggested change.